languages/javascript/keywords/const.md

```diff
@@ -4,9 +4,19 @@ The `const` keyword is used to create a read-only reference to a value.
 ```javascript
 const NAME = "Bob";
 ```
-The value it holds is not immutable, but the variable identifier cannot be reassigned:
+The value it holds is still [mutable](https://developer.mozilla.org/en-US/docs/Glossary/Mutable), but the variable identifier cannot be reassigned:
 ```javascript
 const NAME = "Bob";
 
-const NAME = "Steve"; // This will fail
+NAME = "Steve"; // TypeError: Assignment to constant variable
 ```
+or redeclared:
+```javascript
+const NAME = "Bob"
+
+const NAME = "Sally"
+// SyntaxError: Identifier 'NAME' has already been declared
+```
+Variables declared using the `const` keyword are [block-scoped](https://developer.mozilla.org/en-US/docs/Glossary/Scope), meaning they are only accessible within the current block, and any contained sub-blocks.
```

Comment:
We do not yet have a language specific document on scope and both [here](https://github.com/exercism/v3/tree/1acc146c13dcc5268c30ab40d7e76f35285ef567/languages/javascript/reference#general-concepts) and [here](https://github.com/exercism/v3/tree/1acc146c13dcc5268c30ab40d7e76f35285ef567/languages/javascript/reference#object-oriented-concepts) it refers to the generic one. I recommend creating a new file [here](https://github.com/exercism/v3/tree/1acc146c13dcc5268c30ab40d7e76f35285ef567/languages/javascript/reference/info) called `scope.md`, and linking to both [`/reference/scope.md`](https://github.com/exercism/v3/blob/master/reference/concepts/scope.md), and [mdn](https://developer.mozilla.org/en-US/docs/Glossary/Scope). Add a section _inside this file_ about the Temporal Dead Zone, with another link to [mdn](https://wesbos.com/temporal-dead-zone/).